From 0ce51e4bc6d5094c2c5412c57afb9ab963dc44bd Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 8 Nov 2017 20:51:23 +0000
Subject: [PATCH] Emulate clock_gettime on macOS < 10.12 too

---
 src/tinycthread.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/tinycthread.h b/src/tinycthread.h
index 128b7b6..523f452 100644
--- a/src/tinycthread.h
+++ b/src/tinycthread.h
@@ -112,8 +112,8 @@ freely, subject to the following restrictions:
   #endif
 #endif
 
-/* Workaround for missing clock_gettime (most Windows compilers, afaik) */
-#if defined(_TTHREAD_WIN32_)
+/* Workaround for missing clock_gettime (most Windows compilers, and macOS < 10.12 afaik) */
+#if defined(_TTHREAD_WIN32_) || (defined(__APPLE__) && !defined(CLOCK_MONOTONIC))
 #define _TTHREAD_EMULATE_CLOCK_GETTIME_
 /* Emulate struct timespec */
 struct _ttherad_timespec {
-- 
2.10.1

