From f89eea16e6d1dba6f0b6566c2977c9adbdddc820 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Aug 2016 14:51:51 +0100
Subject: [PATCH 06/45] Qt5Sensors is not required

---
 src/cpp/desktop/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index 561d0f11d6..9bebbcbcb8 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -85,7 +85,6 @@ find_package(Qt5WebChannel REQUIRED)
 find_package(Qt5WebKitWidgets REQUIRED)
 find_package(Qt5PrintSupport REQUIRED)
 find_package(Qt5Quick REQUIRED)
-find_package(Qt5Sensors REQUIRED)
 find_package(Qt5Svg REQUIRED)
 find_package(Qt5Xml REQUIRED)
 find_package(Qt5XmlPatterns REQUIRED)
@@ -312,7 +311,6 @@ if(NOT APPLE)
       WebKitWidgets
       PrintSupport
       Quick
-      Sensors
       Svg
       Xml
       XmlPatterns
-- 
2.14.2

