From 6e90bab14c19be3233708804b52d5e0a03e043f4 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 9 Sep 2018 12:49:38 +0100
Subject: [PATCH 48/52] Server mode also needs diagnostics

---
 src/cpp/CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
index 247e6fad80..c85d69fde6 100644
--- a/src/cpp/CMakeLists.txt
+++ b/src/cpp/CMakeLists.txt
@@ -379,9 +379,11 @@ else()
       # test runner script
       configure_file(rstudio-tests.in ${CMAKE_CURRENT_BINARY_DIR}/rstudio-tests)
 
+      # Server also needs this
+      add_subdirectory(diagnostics)
+
       # add desktop subprojects if we aren't building in server only mode
       if(RSTUDIO_DESKTOP)
-         add_subdirectory(diagnostics)
          if(NOT APPLE)
             add_subdirectory(desktop)
          else()
-- 
2.19.1

