From d1dfbc59860b9030648c2027add8c6911928716c Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 19 Nov 2018 12:16:11 +0000
Subject: [PATCH 52/52] Hack MACOS_VERSION to 10.9

---
 src/cpp/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
index c85d69fde6..4693d5bf21 100644
--- a/src/cpp/CMakeLists.txt
+++ b/src/cpp/CMakeLists.txt
@@ -78,6 +78,7 @@ if(UNIX)
       # CMAKE_SYSTEM_VERSION does not match uname -r, so get the Mac OS
       # version number another way.
       EXECUTE_PROCESS(COMMAND /usr/bin/sw_vers -productVersion OUTPUT_VARIABLE MACOSX_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
+      SET(MACOS_VERSION 10.9)
       message(STATUS "Mac OS X version: ${MACOSX_VERSION}")
       if(RSTUDIO_USE_LIBCXX)
          message(STATUS "C++ Standard Library: libc++")
-- 
2.19.1

