From d8aecfe1adbc68e12d675839f86a65e69cd35aac Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Aug 2016 14:34:20 +0100
Subject: [PATCH 02/40] Qt5Positioning is not required

---
 src/cpp/desktop/CMakeLists.txt |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index f5701bf..10a61e7 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -112,7 +112,6 @@ find_package(Qt5WebEngine REQUIRED)
 find_package(Qt5WebEngineWidgets REQUIRED)
 find_package(Qt5PrintSupport REQUIRED)
 find_package(Qt5Quick REQUIRED)
-find_package(Qt5Positioning REQUIRED)
 find_package(Qt5Sensors REQUIRED)
 find_package(Qt5Svg REQUIRED)
 find_package(Qt5Xml REQUIRED)
@@ -330,7 +329,6 @@ set(RSTUDIO_QTMODULES
     PrintSupport
     Quick
     QuickWidgets
-    Positioning
     Sensors
     Svg
     Xml
-- 
1.7.1

