From ba5d2a67b4050e4a3e0210da2d4a5693b4bc08d3 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 9 Aug 2016 11:49:59 +0100
Subject: [PATCH 02/35] Debugging

---
 src/cpp/desktop/DesktopSessionLauncher.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/cpp/desktop/DesktopSessionLauncher.cpp b/src/cpp/desktop/DesktopSessionLauncher.cpp
index 6d2996870f..d0539d9607 100644
--- a/src/cpp/desktop/DesktopSessionLauncher.cpp
+++ b/src/cpp/desktop/DesktopSessionLauncher.cpp
@@ -106,6 +106,7 @@ Error SessionLauncher::launchFirstSession()
    logEnvVar("PATH");
    logEnvVar("HOME");
    logEnvVar("R_USER");
+    RUN_DIAGNOSTICS_LOG("\nSessionLauncher::launchFirstSession ..\n");
 
    // launch the process
    Error error = launchSession(argList, &pRSessionProcess_);
@@ -328,6 +329,9 @@ Error SessionLauncher::launchSession(const QStringList& argList,
    if (error)
       LOG_ERROR(error);
 
+   RUN_DIAGNOSTICS_LOG("\nSessionLauncher::launch: " + sessionPath_.absolutePath() + "\n");
+
+
    return  parent_process_monitor::wrapFork(
          boost::bind(launchProcess,
                      sessionPath_.absolutePath(),
-- 
2.14.3 (Apple Git-98)

