From acf0af1cf7709765b00ee85c2ffa219e2ccb9d88 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 17 Jul 2018 07:33:52 -0700
Subject: [PATCH 03/41] Qt5Sensors is not required

---
 src/cpp/desktop/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index fba21914d0..f8d720d5bd 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -123,7 +123,6 @@ find_package(Qt5WebEngineWidgets REQUIRED)
 find_package(Qt5PrintSupport REQUIRED)
 find_package(Qt5Quick REQUIRED)
 find_package(Qt5QuickWidgets REQUIRED)
-find_package(Qt5Sensors REQUIRED)
 find_package(Qt5Svg REQUIRED)
 find_package(Qt5Xml REQUIRED)
 find_package(Qt5XmlPatterns REQUIRED)
@@ -342,7 +341,6 @@ set(RSTUDIO_QTMODULES
     Qt5::PrintSupport
     Qt5::Quick
     Qt5::QuickWidgets
-    Qt5::Sensors
     Qt5::Svg
     Qt5::Xml
     Qt5::XmlPatterns
-- 
2.20.1

