From 274e1178910e0e29f746aed217a3c692e032b4ed Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Aug 2016 14:51:51 +0100
Subject: [PATCH 03/40] Qt5Sensors is not required

---
 src/cpp/desktop/CMakeLists.txt |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index 10a61e7..76f85ab 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -112,7 +112,6 @@ find_package(Qt5WebEngine REQUIRED)
 find_package(Qt5WebEngineWidgets REQUIRED)
 find_package(Qt5PrintSupport REQUIRED)
 find_package(Qt5Quick REQUIRED)
-find_package(Qt5Sensors REQUIRED)
 find_package(Qt5Svg REQUIRED)
 find_package(Qt5Xml REQUIRED)
 find_package(Qt5XmlPatterns REQUIRED)
@@ -329,7 +328,6 @@ set(RSTUDIO_QTMODULES
     PrintSupport
     Quick
     QuickWidgets
-    Sensors
     Svg
     Xml
     XmlPatterns
-- 
1.7.1

