From 85ea0bf1778dc17233c56c9d5e16134ecfa5e7be Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Aug 2016 14:34:20 +0100
Subject: [PATCH 03/42] Qt5Positioning is not required

---
 src/cpp/desktop/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index 2fa56cfba3..21e160f395 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -112,7 +112,6 @@ find_package(Qt5WebEngine REQUIRED)
 find_package(Qt5WebEngineWidgets REQUIRED)
 find_package(Qt5PrintSupport REQUIRED)
 find_package(Qt5Quick REQUIRED)
-find_package(Qt5Positioning REQUIRED)
 find_package(Qt5Sensors REQUIRED)
 find_package(Qt5Svg REQUIRED)
 find_package(Qt5Xml REQUIRED)
@@ -330,7 +329,6 @@ set(RSTUDIO_QTMODULES
     PrintSupport
     Quick
     QuickWidgets
-    Positioning
     Sensors
     Svg
     Xml
-- 
2.14.3 (Apple Git-98)

