From 3af56ad338029f286f4b9039d9cffbd559783b4c Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Aug 2016 14:51:51 +0100
Subject: [PATCH 04/42] Qt5Sensors is not required

---
 src/cpp/desktop/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index 21e160f395..f00c7a5f3d 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -112,7 +112,6 @@ find_package(Qt5WebEngine REQUIRED)
 find_package(Qt5WebEngineWidgets REQUIRED)
 find_package(Qt5PrintSupport REQUIRED)
 find_package(Qt5Quick REQUIRED)
-find_package(Qt5Sensors REQUIRED)
 find_package(Qt5Svg REQUIRED)
 find_package(Qt5Xml REQUIRED)
 find_package(Qt5XmlPatterns REQUIRED)
@@ -329,7 +328,6 @@ set(RSTUDIO_QTMODULES
     PrintSupport
     Quick
     QuickWidgets
-    Sensors
     Svg
     Xml
     XmlPatterns
-- 
2.14.3 (Apple Git-98)

