From 4bd49011086d02c1a36df89e808e82a03a8bb21d Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Wed, 10 Aug 2016 14:34:20 +0100
Subject: [PATCH 06/44] Qt5Positioning is not required

---
 src/cpp/desktop/CMakeLists.txt | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index 1e1117d2a6..561d0f11d6 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -85,7 +85,6 @@ find_package(Qt5WebChannel REQUIRED)
 find_package(Qt5WebKitWidgets REQUIRED)
 find_package(Qt5PrintSupport REQUIRED)
 find_package(Qt5Quick REQUIRED)
-find_package(Qt5Positioning REQUIRED)
 find_package(Qt5Sensors REQUIRED)
 find_package(Qt5Svg REQUIRED)
 find_package(Qt5Xml REQUIRED)
@@ -313,7 +312,6 @@ if(NOT APPLE)
       WebKitWidgets
       PrintSupport
       Quick
-      Positioning
       Sensors
       Svg
       Xml
-- 
2.14.1

