From 8d56c98df5204418f5070bb0c67bca5df0bff536 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 22 Oct 2016 18:37:40 +0100
Subject: [PATCH 06/35] Boost_USE_STATIC_LIBS when DEFINED ENV{CONDA_BUILD}

---
 src/cpp/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cpp/CMakeLists.txt b/src/cpp/CMakeLists.txt
index 0fbd107565..7917ca9aec 100644
--- a/src/cpp/CMakeLists.txt
+++ b/src/cpp/CMakeLists.txt
@@ -152,7 +152,7 @@ endif()
 # determine whether we should statically link boost. we always do this
 # unless we are building a non-packaged build on linux (in which case
 # boost dynamic libraries are presumed to be installed on the system ldpath)
-if(APPLE OR WIN32 OR RSTUDIO_PACKAGE_BUILD)
+if(APPLE OR WIN32 OR RSTUDIO_PACKAGE_BUILD OR DEFINED ENV{CONDA_BUILD})
    set(Boost_USE_STATIC_LIBS ON)
 endif()
 
-- 
2.14.3 (Apple Git-98)

