From f7454cd40e9f2ddaec69c7e4ccec8788c76f4de2 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 24 Mar 2018 17:50:35 +0000
Subject: [PATCH 26/38] Set CMP0025 for custom clang on macOS

Needs CMake 3.3 at least
---
 CMakeCompiler.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeCompiler.txt b/CMakeCompiler.txt
index 0a63223cff..c854cc6848 100644
--- a/CMakeCompiler.txt
+++ b/CMakeCompiler.txt
@@ -104,3 +104,4 @@ if(WIN32)
   set(CMAKE_COLOR_MAKEFILE OFF)
 endif()
 
+cmake_policy(SET CMP0025 NEW)
-- 
2.23.0

