From e25ff49324b739ee49666b2c082bc668510b5040 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 25 Mar 2018 16:11:37 +0000
Subject: [PATCH 27/38] Add missing std::mutex header

---
 src/cpp/desktop/DesktopWebPage.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cpp/desktop/DesktopWebPage.cpp b/src/cpp/desktop/DesktopWebPage.cpp
index 42f12c5650..f68a0dcf11 100644
--- a/src/cpp/desktop/DesktopWebPage.cpp
+++ b/src/cpp/desktop/DesktopWebPage.cpp
@@ -16,6 +16,7 @@
 #include "DesktopWebPage.hpp"
 
 #include <boost/algorithm/string.hpp>
+#include <mutex>
 #include <boost/thread/once.hpp>
 
 #include <core/http/URL.hpp>
-- 
2.23.0

