From 6bbcb03dbb121fa542aa32257a28c580ad01908a Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 24 Mar 2018 17:30:05 +0000
Subject: [PATCH 28/40] Use find_program to find qmake as a last-ditch fallback

---
 src/cpp/desktop/CMakeLists.txt |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index be33b1d..7feb5ef 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -88,6 +88,10 @@ else()
    endif()
 endif()
 
+if(NOT QT_QMAKE_EXECUTABLE)
+   find_program(QT_QMAKE_EXECUTABLE qmake)
+endif()
+
 if(QT_QMAKE_EXECUTABLE)
    message(STATUS "Found Qt: ${QT_QMAKE_EXECUTABLE}")
 endif()
-- 
1.7.1

