From b39c86de9bba934203e520def3d18295ff6e89a8 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 24 Mar 2018 17:30:05 +0000
Subject: [PATCH 28/42] Use find_program to find qmake as a last-ditch fallback

---
 src/cpp/desktop/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index d6a21e29b9..ddb012b4f1 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -88,6 +88,10 @@ else()
    endif()
 endif()
 
+if(NOT QT_QMAKE_EXECUTABLE)
+   find_program(QT_QMAKE_EXECUTABLE qmake)
+endif()
+
 if(QT_QMAKE_EXECUTABLE)
    message(STATUS "Found Qt: ${QT_QMAKE_EXECUTABLE}")
 endif()
-- 
2.15.1 (Apple Git-101)

