From 364a7db1bb44f9015cc95f61a341efbb1c3728e9 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sat, 24 Mar 2018 17:30:05 +0000
Subject: [PATCH 28/41] Use find_program to find qmake as a last-ditch fallback

---
 src/cpp/desktop/CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/cpp/desktop/CMakeLists.txt b/src/cpp/desktop/CMakeLists.txt
index 7b974552e8..c7899cfd33 100644
--- a/src/cpp/desktop/CMakeLists.txt
+++ b/src/cpp/desktop/CMakeLists.txt
@@ -96,6 +96,10 @@ else()
    endif()
 endif()
 
+if(NOT QT_QMAKE_EXECUTABLE)
+   find_program(QT_QMAKE_EXECUTABLE qmake)
+endif()
+
 if(QT_QMAKE_EXECUTABLE)
    message(STATUS "Found Qt: ${QT_QMAKE_EXECUTABLE}")
 else()
-- 
2.20.1

