From b2f0153b8c2b634ce25f4103273c69f9abdc6529 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 25 Mar 2018 16:11:37 +0000
Subject: [PATCH 30/40] Add missing std::mutex header

---
 src/cpp/desktop/DesktopWebPage.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cpp/desktop/DesktopWebPage.cpp b/src/cpp/desktop/DesktopWebPage.cpp
index a4d056d..8639be4 100644
--- a/src/cpp/desktop/DesktopWebPage.cpp
+++ b/src/cpp/desktop/DesktopWebPage.cpp
@@ -16,6 +16,7 @@
 #include "DesktopWebPage.hpp"
 
 #include <boost/algorithm/string.hpp>
+#include <mutex>
 
 #include <core/Thread.hpp>
 
-- 
1.9.5 (Apple Git-50.3)

