From 93a08df7fc86dad8cbc4274084af329f0cd1f0c9 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 25 Mar 2018 16:11:37 +0000
Subject: [PATCH 30/41] Add missing std::mutex header

---
 src/cpp/desktop/DesktopWebPage.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cpp/desktop/DesktopWebPage.cpp b/src/cpp/desktop/DesktopWebPage.cpp
index e87c0c5160..ac1b0cc88f 100644
--- a/src/cpp/desktop/DesktopWebPage.cpp
+++ b/src/cpp/desktop/DesktopWebPage.cpp
@@ -16,6 +16,7 @@
 #include "DesktopWebPage.hpp"
 
 #include <boost/algorithm/string.hpp>
+#include <mutex>
 
 #include <core/Thread.hpp>
 
-- 
2.20.1

