From 2365eb3643ba90fe1931f66a7492cb8a8c1247ad Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Tue, 27 Mar 2018 04:58:02 -0700
Subject: [PATCH 31/41] Force overwrite when unzipping dependencies on Windows

---
 dependencies/windows/install-dependencies.cmd | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dependencies/windows/install-dependencies.cmd b/dependencies/windows/install-dependencies.cmd
index 20acd99aa8..389de9b7ce 100644
--- a/dependencies/windows/install-dependencies.cmd
+++ b/dependencies/windows/install-dependencies.cmd
@@ -5,7 +5,7 @@ setlocal
 set PATH=%CD%\tools;%PATH%
 
 set WGET_ARGS=-c --no-check-certificate
-set UNZIP_ARGS=-q
+set UNZIP_ARGS=-q -o
 
 set BASEURL=https://s3.amazonaws.com/rstudio-buildtools/
 set GIN_FILE=gin-2.1.2.zip
-- 
2.20.1

