From 848384dcb5e2705cd13cf72aaeb7e804909dc1ae Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 25 Mar 2018 16:11:37 +0000
Subject: [PATCH 31/42] Add missing std::mutex header

---
 src/cpp/desktop/DesktopWebPage.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/cpp/desktop/DesktopWebPage.cpp b/src/cpp/desktop/DesktopWebPage.cpp
index cda7f397b4..5ef83e34a4 100644
--- a/src/cpp/desktop/DesktopWebPage.cpp
+++ b/src/cpp/desktop/DesktopWebPage.cpp
@@ -16,6 +16,7 @@
 #include "DesktopWebPage.hpp"
 
 #include <boost/algorithm/string.hpp>
+#include <mutex>
 
 #include <core/Thread.hpp>
 
-- 
2.15.1 (Apple Git-101)

