From e6ef010b08cf5928ffa9a491a3743b4ade038dd4 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 25 Mar 2018 16:11:37 +0000
Subject: [PATCH 37/37] Add missing std::mutex header

---
 src/cpp/desktop/DesktopWebPage.cpp |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/cpp/desktop/DesktopWebPage.cpp b/src/cpp/desktop/DesktopWebPage.cpp
index 6f248da..09b6db9 100644
--- a/src/cpp/desktop/DesktopWebPage.cpp
+++ b/src/cpp/desktop/DesktopWebPage.cpp
@@ -16,6 +16,7 @@
 #include "DesktopWebPage.hpp"
 
 #include <boost/algorithm/string.hpp>
+#include <mutex>
 
 #include <core/Thread.hpp>
 
-- 
1.7.1

