diff -urN work/src/cpp/CMakeLists.txt.orig work/src/cpp/CMakeLists.txt
--- work/src/cpp/CMakeLists.txt.orig	2017-10-18 03:26:57.000000000 -0700
+++ work/src/cpp/CMakeLists.txt	2017-10-18 03:49:42.000000000 -0700
@@ -94,6 +94,11 @@
             set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -stdlib=libstdc++")
          endif()
       endif()
+      # Apple have a macro in Cocoa somewhere called check()
+      # which conflicts with the Boost assert macros.
+      # See: https://github.com/facebook/osquery/pull/3613/files
+      # and: https://svn.boost.org/trac10/ticket/6219
+      add_definitions(-D__ASSERT_MACROS_DEFINE_VERSIONS_WITHOUT_UNDERSCORES=0)
    endif()
 
    # gcc hardending options (see: http://wiki.debian.org/Hardening)
